application/src/objects/obstacles/abstract-dynamic-obstacle.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var Phaser = require('phaser');
+
+var AbstractDynamicObstacle = function(state, x, y, key, collisionGroup)
+{
+    Phaser.Sprite.apply(this, [state.game, x, y, key]);
+
+    this.createPhysicsBody(state);
+
+    this.body.setCollisionGroup(collisionGroup);
```

Comment:
I left this in the state object for cars. Not sure where it belongs.

I had initially accepted a group arg in the CarSprite constructor because http://docs.phaser.io/Phaser.GameObjectFactory.html#sprite but now I realize that the arg doesn't actually exist in the Sprite constructor (http://docs.phaser.io/Sprite.js_.html#sunlight-1-line-23).